app/helpers/content_pages_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ContentPagesHelper
+  def get_all_content_pages
```

Comment:
Using a description of the action used to create an object to name it, isn't a very Rails way of doing things. It's better practice to have object names describe the object they returns. So this would be a better name:
```ruby
def top_level_content_pages_by_position
```
However, I'd also question why this is being done in a helper method, which should really only be used to decorate/modify view behaviour. I'd suggest that a better option would be to create a new scope in the content page model:
```ruby
scope :top_level_by_position, -> { top_level.order_by_position }
```
Then in the views, use `ContentPage.top_level_by_position` where it is needed.

Reply:
I'd also just chain it, that's also perfectly acceptable `ContentPage.top_level.order_by_position`

Reply:
i've made this suggested change